Networking/NetworkingTests/Responses/countries.json

```diff
@@ -0,0 +1,328 @@
+                    "name": "New York"
+                },
+                {
+                    "code": "NC",
+                    "name": "North Carolina"
+                },
+                {
+                    "code": "ND",
+                    "name": "North Dakota"
+                },
+                {
+                    "code": "OH",
+                    "name": "Ohio"
+                },
+                {
+                    "code": "OK",
+                    "name": "Oklahoma"
+                },
+                {
+                    "code": "OR",
+                    "name": "Oregon"
+                },
+                {
+                    "code": "PA",
+                    "name": "Pennsylvania"
+                },
+                {
+                    "code": "RI",
+                    "name": "Rhode Island"
+                },
+                {
+                    "code": "SC",
+                    "name": "South Carolina"
+                },
+                {
+                    "code": "SD",
+                    "name": "South Dakota"
+                },
+                {
+                    "code": "TN",
+                    "name": "Tennessee"
+                },
+                {
+                    "code": "TX",
+                    "name": "Texas"
+                },
+                {
+                    "code": "UT",
+                    "name": "Utah"
+                },
+                {
+                    "code": "VT",
+                    "name": "Vermont"
+                },
+                {
+                    "code": "VA",
+                    "name": "Virginia"
+                },
+                {
+                    "code": "WA",
+                    "name": "Washington"
+                },
+                {
+                    "code": "WV",
+                    "name": "West Virginia"
+                },
+                {
+                    "code": "WI",
+                    "name": "Wisconsin"
+                },
+                {
+                    "code": "WY",
+                    "name": "Wyoming"
+                },
+                {
+                    "code": "AA",
+                    "name": "Armed Forces (AA)"
+                },
+                {
+                    "code": "AE",
+                    "name": "Armed Forces (AE)"
+                },
+                {
+                    "code": "AP",
+                    "name": "Armed Forces (AP)"
+                }
+            ],
+            "_links": {
+                "self": [
+                    {
+                        "href": "https://mywootesting.mystagingwebsite.com/wp-json/wc/v3/data/countries/us"
+                    }
+                ],
+                "collection": [
+                    {
+                        "href": "https://mywootesting.mystagingwebsite.com/wp-json/wc/v3/data/countries"
+                    }
+                ]
+            }
+        }
+    ]
+}
```

Comment:
Some countries don't have states or provinces. For completeness, the unit test should include at least one such country, e.g. I think Austria (AT) might work

Reply:
Good idea 👍 done